<ui version="4.0" >
 <class>CoverDialogBase</class>
 <widget class="QWidget" name="CoverDialogBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Cover Manager</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>11</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="K3ListView" name="m_artists" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>7</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>164</width>
       <height>0</height>
      </size>
     </property>
     <property name="resizeMode" >
      <enum>Q3ListView::LastColumn</enum>
     </property>
     <property name="shadeSortColumn" >
      <bool>false</bool>
     </property>
     <column>
      <property name="text" >
       <string>Artist</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="K3IconViewSearchLine" native="1" name="m_searchLine" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>5</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="CoverIconView" native="1" name="m_covers" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>7</hsizetype>
         <vsizetype>7</vsizetype>
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>K3ListView</class>
   <extends>Q3ListView</extends>
   <header>k3listview.h</header>
  </customwidget>
  <customwidget>
   <class>K3IconViewSearchLine</class>
   <extends>QWidget</extends>
   <header>k3iconviewsearchline.h</header>
  </customwidget>
  <customwidget>
   <class>CoverIconView</class>
   <extends>QWidget</extends>
   <header>covericonview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >k3listview.h</include>
  <include location="local" >k3iconviewsearchline.h</include>
  <include location="local" >covericonview.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>m_artists</sender>
   <signal>clicked(Q3ListViewItem*)</signal>
   <receiver>CoverDialogBase</receiver>
   <slot>slotArtistClicked(Q3ListViewItem*)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_covers</sender>
   <signal>contextMenuRequested(Q3IconViewItem*,QPoint)</signal>
   <receiver>CoverDialogBase</receiver>
   <slot>slotContextRequested(Q3IconViewItem*,QPoint)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
